lib/tasks/load_campaign_example.rake

```diff
@@ -1,110 +1,18 @@
-require "example_campaign_data"
+require "load_example_campaign"
 
 desc "Load campaign example file into db"
-task :load_campaign_example, [:example_file] => :environment do |_task, args|
-  new_campaign = ENV.fetch("new_campaign", false).in? %w[true 1 yes]
+task :load_campaign_example,
+     %i[example_file new_campaign] => :environment do |_task, args|
+  new_campaign =
```

Comment:
Strictly speaking this bit is a stowaway from #532, but it's not causing harm and isn't worth the time to be split off, unless it turns out to be controversial.